sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -332,6 +332,11 @@ def test_upretty_modifiers():
     assert upretty( Symbol('x__dot') ) == u('x__dot')
 
 
+def test_pretty_Cycle():
+    from sympy.combinatorics.permutations import Cycle
+    assert pretty(Cycle(1, 2)) == '([1, 2],)'
```

Comment:
Many classes have two printers (in addition to pretty printer). One in repr.py and another in str.py. For example, one ``_print_Add`` will show the structure: ``Add(x, y)`` while the other one prints the standard expression ``x + y``. There is only one (non-pretty) printer for ``Cycle`` but maybe we could have two. One with ``Cycle`` to allow recreating the object and the other one showing the usual cycle representation of a permutation. The latter could also omit the commas (which, I think, looks better).